### ISSUE TITLE
Miki will ban anyone even when person executing the command doesn't have perms to do so?

### ISSUE THREAD

**Issue by vekst** [veldlabs/miki-bot#525]
Basically, on a server I moderate, my friend decided to do >ban (user) on a guy just as a joke, as she doesn't have any mod perms whatsoever. However, Miki banned the guy with no hassle. I assume if one does >kick (user) without having any of the right perms, it'll work as well.

**Comment by velddev** [comment]
This only works if the user has the `ManageRoles` permissions, as this determines someone is an admin on the server in Miki's eyes.

**Comment by vekst** [comment]
Odd, I don't believe any of their roles have the ManageRoles perms but I'll double check anyways.

**Comment by velddev** [comment]
Any update on this?

**Comment by vekst** [comment]
Oh apologies, forgot to update it. They did not have any roles that gave ManageRoles. So after this, I decided to do some testing with my main and alt to see what else it could've been. Even as an owner of the server I tested in, having all perms enabled possible (including ManageRoles), I cannot kick my alt that has perms or anything whatsoever. Miki also has all perms so now I'm just completely confused.
